### PR TITLE
[Windows] Update cabal pester test

### PR DESCRIPTION
**Pull request by al-cheb**
# Description

```
2022-02-22T00:34:39.3388641Z  vhd:   [-] cabal config was modified and exists 166ms (164ms|2ms)
2022-02-22T00:34:39.3399886Z  vhd:    Expected path 'C:\cabal' to exist, but it did not exist.
2022-02-22T00:34:39.3400920Z  vhd:    at $env:CABAL_DIR | Should -Exist, C:\image\Tests\Haskell.Tests.ps1:53
2022-02-22T00:34:39.3405661Z  vhd:    at <ScriptBlock>, C:\image\Tests\Haskell.Tests.ps1:53
```

https://gitlab.haskell.org/haskell/ghcup-hs/-/commit/2c583bcae9e333693b9bd8fc353bb0dcb2af71df - Fix NoAdjustCabalConfig on windows

Fix:
- Add test to check cabal folder

#### Related issue:
https://github.com/actions/virtual-environments/issues/5119

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by al-cheb** [comment]
@hasufell, Could you please review?

**Review by miketimofeev — APPROVED** [review]

**Review by Chaouki-Tiouassiouine — APPROVED** [review]

File: images/win/scripts/Tests/Haskell.Tests.ps1 [hunk 1]
```diff
@@ -49,9 +49,8 @@ Describe "Haskell" {
         "cabal --version" | Should -ReturnZeroExitCode
     }
 
-    It "cabal config was modified and exists" {
-        $env:CABAL_DIR | Should -Exist
-        "cabal user-config diff" | Should -ReturnZeroExitCode
+    It "cabal folder does not exist" {
+        $env:CABAL_DIR | Should -Not -Exist
```
Comment (hasufell) [inline images/win/scripts/Tests/Haskell.Tests.ps1:53]: I think we still set this env var... the main difference now is that `cabal user-config diff` should return an **empty string**.

Comment (al-cheb) [reply images/win/scripts/Tests/Haskell.Tests.ps1:53]: @hasufell , `Expected path 'C:\cabal' to exist, but it did not exist.` -  But this path doesn't exist after the changes.

Comment (hasufell) [reply images/win/scripts/Tests/Haskell.Tests.ps1:53]: Ah right... but can we check whether the env var is set?

Comment (al-cheb) [reply images/win/scripts/Tests/Haskell.Tests.ps1:53]: Thank you. Done.

**Review by hasufell — APPROVED** [review]

**Comment by krokofant** [comment]
image
image

Are there different kinds of reviews or is this message just confusing? Since hasufell has reviewed it, why is it still waiting for _approving review_? 🤔

**Comment by miketimofeev** [comment]
> image image
> 
> Are there different kinds of reviews or is this message just confusing? Since hasufell has reviewed it, why is it still waiting for _approving review_? 🤔

at least one reviewer **with write access** that's the main difference

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**